newspaper/network.py

```diff
@@ -71,7 +71,10 @@ def get_html_2XX_only(url, config=None, response=None):
     return html
 
 
-def _get_html_from_response(response):
+def _get_html_from_response(response, config=None):
+    for disabled_type in config.ignored_content_types_defaults:
+        if disabled_type in response.headers.get('content-type'):
+            return config.ignored_content_types_defaults[disabled_type]
```

Comment:
Why return the sentinel empty pdf constant `%PDF-` instead of just an empty string? I feel returning special values like this will confuse the API, especially for a low-level function such as `_get_html_from_response`

Reply:
Thanks for your feedback, I agree this piece of code looks quite unexpected and ugli. Unfortunately I didn't found any standard way to understand what was the content type of the article from the client of the library. So what we are doing as a client of the library now is just checking the first 5 letters to be equal to this string. Is there any better way of getting the type of the article?